target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -1540,14 +1581,26 @@ static int rtl8367b_detect(struct rtl8366_smi *smi)
 		return ret;
 	}
 
-	switch (chip_ver) {
-	case 0x1000:
-		chip_name = "8367RB";
-		break;
-	case 0x1010:
-		chip_name = "8367R-VB";
-		break;
-	default:
+	if (smi->flags & RTL8367B_FLAG_MATCH_CHIP_NUM) {
+		switch (chip_num) {
+		case 0x0276:
+		case 0x0597:
+		case 0x6367:
+			chip_name = "8367C";
```

Comment:
This is not a chip name. rtl8367c is the vendor api version (as there are rtl8367b, rtl8367d). Please, use the corresponding chip name for each chip_num. You need to match chip_id with chip_ver to match the chip_name. You can check some known models at https://github.com/torvalds/linux/blob/master/drivers/net/dsa/realtek/rtl8365mb.c#L523